var/spack/repos/builtin/packages/nalu-wind/package.py

```diff
@@ -127,6 +127,8 @@ class NaluWind(CMakePackage, CudaPackage, ROCmPackage):
     conflicts("^trilinos+cuda", when="~cuda")
     conflicts("^trilinos+rocm", when="~rocm")
     conflicts("+shared", when="+trilinos-solvers")
+    conflicts("openfast@4.0.0", msg="OpenFAST 4.0.0 contains a bug. Use OpenFAST >= 4.0.2.")
+    conflicts("openfast@4.0.1", msg="OpenFAST 4.0.1 contains a bug. Use OpenFAST >= 4.0.2.")
```

Comment:
Perhaps deprecate this version in Openfast so the concretizer wont pick it then? Also I think you should be able to combine these two conflicts into one by setting the version as a range

Reply:
I toyed with that idea but then thought it might be confusing. Someone might think later that they were accidentally left out and add them back in. I would have to leave an annoying comment in there. 

I combined the conflicts. Good idea.

Reply:
I think Phil means https://github.com/spack/spack/blob/5e981797f5ce6300b7aa1cab85683dc0ee00cbc2/var/spack/repos/builtin/packages/llvm/package.py#L73

Reply:
oh that's cool. Can definitely do that